student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'Person'
+
+class Student < Person
+  def initialize(age:, classroom:, name: 'Unknown', parent_permission: true)
+    super(name: name, parent_permission: parent_permission, age: age)
```

Comment:
- [ ] Kindly make sure to remove the repeated values in the parent constructor, this will make your code cleaner

```suggestion
    super(name, parent_permission, age)
```